src/oemof/tabular/facades.py

```diff
@@ -1578,7 +1578,33 @@ def __init__(self, *args, **kwargs):
 
         self.marginal_cost = kwargs.get("marginal_cost")
```

Comment:
Dispatchable has a default value for `marginal_cost`. Should we have it here as well?

```suggestion
        self.marginal_cost = kwargs.get("marginal_cost", 0)
```